File: Python/sumOfArray.py
#Method 1:
def sumOfArray(arr):
    total = 0
    for num in arr:
        total += num
    return total

arr = [1, 2, 3, 4, 5]
print("Sum of array:", sumOfArray(arr))


#Method 2:
def sumOfArray(arr):
    return sum(arr)

arr = [1, 2, 3, 4, 5]
print("Sum of array:", sumOfArray(arr))

File: Java/sumOfArray.java
public class sumOfArray {
    public static int sumArray(int arr[]){
        int sum = 0;
        for(int i = 0; i < arr.length; i++){
            sum = sum + arr[i];
        }

        return sum;
    }
    public static void main(String[] args) {
        int arr[] = {1, 2, 3, 4, 5};

        System.out.print("Sum of array is: " + sumArray(arr));
    }
}
